src/app/pages/CreateConnectorPage/StepConfigurator.tsx

```diff
@@ -155,7 +181,13 @@ export const ConfiguratorStep: FunctionComponent = () => {
           ? t(configurationSteps[activeConfigurationStep])
           : t('connectorSpecific')
       }
-      description={t('configurationStepDescription')}
+      description={
+        hasCustomConfigurator ? (
+          <ConfiguratorCustomStepDescription actor={configuratorRef} />
```

Comment:
I'd recommend making a custom hook to access the machine and it's state.  [Like here](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/src/app/components/CreateConnectorWizard/CreateConnectorWizardContext.tsx#L321-L322) except you get the `configuratorRef` and then you use `useSelector` to access the active step and then return the right text string.

Reply:
updated